Makefile

```diff
@@ -0,0 +1,78 @@
+# Copyright 2021 The Fission Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+.DEFAULT_GOAL := check
+
+# Platforms to build in multi-architecture images.
+PLATFORMS ?= linux/amd64,linux/arm64,linux/arm/v7
+
+# Repository prefix and tag to push multi-architecture images to.
+REPO ?= fission
+TAG ?= dev
+
+verify-builder: 
+	@./buildx.sh 
+
+multiarch-binary: verify-builder multiarch-binary-env multiarch-binary-builder
+
+multiarch-go: verify-builder multiarch-go-env multiarch-go-builder
+
+multiarch-python: verify-builder multiarch-python-env multiarch-python-builder
+
+multiarch-nodejs: verify-builder multiarch-nodejs-env multiarch-nodejs-builder
+
+multiarch-perl: verify-builder multiarch-perl-env 
+
+multiarch-php: verify-builder multiarch-php-env  multiarch-php-builder
+
+multiarch-ruby: verify-builder multiarch-ruby-env multiarch-ruby-builder
+
+multiarch-binary-env: 
+	docker buildx build --platform=$(PLATFORMS) -t $(REPO)/binary-env:$(TAG) --push -f binary/Dockerfile . 
```

Comment:
Please pass directory name as docker context instead of `.`

Same for all commands below.